runtime/lib_ksils12.c

```diff
@@ -1633,14 +1795,14 @@ void *signer_thread(void *arg) {
 
 		/* in case there are no items go around*/
 		if (ProtectedQueue_count(ctx->signer_queue) == 0) {
-			process_requests_async(ctx, ksi_ctx, as, ksiFile);
+			process_requests_async(ctx, ksi_ctx, as);
 			continue;
 		}
 
 		/* process signing requests only if there is an open signature file */
-		if(ksiFile != NULL) {
+		if(ksiFileCount > 0) {
```

Comment:
could the condition be changed to `ctx->ksiCount`

Reply:
It can, but I like to show that it is not a boolean.